src/outbackcdx/ChangePollingThread.java

```diff
@@ -42,6 +42,9 @@
         } catch (RocksDBException e) {
             System.out.println("Received rocks db exception while looking up the value of the key " + SEQ_NUM_KEY + " locally");
             e.printStackTrace();
+        } catch (Exception e) {
```

Comment:
Catching NullPointerException is frowned upon in Java - it would be normal practice to check for null explicitly here to set the default. :-)

If sequenceNumber is corrupt (i.e. not a number) or there's some error reading the index I think we should probably crash out, rather than assuming 0 and continuing by apply the full update history to a probably-broken index.